### PR TITLE
added documentation files for DroneCore.Suite by Airvolute

### PR DESCRIPTION
**Pull request by lukaspalkovic**
Added documents for new product intended to be listed in companion computers. https://ardupilot.org/copter/docs/common-autopilots.html It has not been prepared in that site yet. I want to do it in next step after I check the documentation site itself works well there.

### CODE REVIEW

File: common/source/docs/common-airvolute-DroneCore-Suite.rst [hunk 1]
```diff
@@ -0,0 +1,134 @@
+.. _common-airvolute-DroneCore-Suite:
```
Comment (Hwurzburg) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:1]: we will need to add this in common-autopilots.rst  as a TOC entry....

File: common/source/docs/common-airvolute-DroneCore-Suite.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+-  Cube flight controller
+-  Internal 5V power supply
+-  Power selector
+-  Peripheral connectors
+-  USB hub
+
+
+**DroneCore.Power** - bottom board with power electronic containing
+
+-  4 FOC UAVCAN ESCs - 20A permanent / 40A peak current
```
Comment (Hwurzburg) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:27]:
```suggestion
-  4 FOC UAVCAN ESCs - 20A permanent / 40A peak current
```
```suggestion
-  4 FOC UAVCAN ESCs - 20A continuous / 40A peak current
```

Comment (Hwurzburg) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:27]:
```suggestion
-  4 FOC UAVCAN ESCs - 20A permanent / 40A peak current
```
```suggestion
-  4 FOC UAVCAN ESCs - 20A continuous / 40A peak current
```

File: common/source/docs/common-airvolute-DroneCore-Suite.rst [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+Various software libraries (coming soon) further enrich the possibilities for the developer relieving him from complex configuration and programming tasks.
+
+DroneCore.Suite provides easy to use ROS based software stack supporting development of autonomous applications.
+
+Out of the box device is configured to fly a quadcopter (though flight controller calibration is needed anyway).
+
+---------------------------
+Top view
+---------------------------
+.. image:: ../../../images/airvolute_DCS_top_view_9219.jpg
```
Comment (Hwurzburg) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:49]: not the correct file name

Comment (Hwurzburg) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:49]: not the correct file name

File: common/source/docs/common-airvolute-DroneCore-Suite.rst [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+-  4 x FOC UAVCAN ESCs – 40A peak / 20A permanent
+-  power sensor, SMBUS,
+-  LED Driver for 4x WS2812B strips
+
+
+---------------------------
+Block Schematic
+---------------------------
+.. image:: ../../../images/airvolute_DroneCore.Pilot_block_diagram.svg
+
```
Comment (Hwurzburg) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:112]: add  target link to show full view when double clicked:
```suggestion
```suggestion
    :target: ../../_images/airvolute_DroneCore.Pilot_block_diagram.svg
```
```

Comment (Hwurzburg) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:112]: add  target link to show full view when double clicked:
```suggestion
```suggestion
    :target: ../../_images/airvolute_DroneCore.Pilot_block_diagram.svg
```
```

File: common/source/docs/common-airvolute-DroneCore-Suite.rst [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+`Product website`_
+
+.. _Product website: https://www.airvolute.com/product/dronecore/
+`Documentation`_
+
+.. _Documentation: https://docs.airvolute.com/airvolute-dronecore.suite
+`Where to buy`_
+
+.. _Where to buy: https://www.airvolute.com/contact/
```
Comment (Hwurzburg) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:132]: normally a separate section header, but I am fine with this as is

Comment (Hwurzburg) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:132]: normally a separate section header, but I am fine with this as is

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: common/source/docs/common-airvolute-DroneCore-Suite.rst [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+-  Integrated DC/DC converter for control circuits
+-  Redundant power supply with power good monitoring for control unit
+-  Current protected peripheral connectors
+-  4 x FOC UAVCAN ESC 40A, featuring motor identification and motor
+   diagnostics
+-  Power sensor, SMBUS
+
+**Ardupilot (Cube) Connections**
+
+-  PPM input
```
Comment (nkruzan) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:77]: Only supports PPM RC?

File: common/source/docs/common-airvolute-DroneCore-Suite.rst [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+---------------------------
+Block Schematic
+---------------------------
+.. image:: ../../../images/airvolute_DroneCore.Pilot_block_diagram.svg
+
+   
+
+**DroneCore.Pilot** is based on Cube Orange - widely used flight controller by Hex.aero running on Ardupilot (or optionally PX4 flight stack) and Nvidia Jetson Xavier NX. Their rich interfaces allow connecting of most of sensor and other peripherals for drones available on the market.
+
+**DroneCore.Power** contains 4 FOC ESCs with motor identification feature able to be configured for all common BLDC/PMSM drone motors on the market and telemetry interface providing real time data from the flight to the flight controller. Furthermore it provides battery voltage and current measurements including separate cell voltages.
```
Comment (nkruzan) [inline common/source/docs/common-airvolute-DroneCore-Suite.rst:117]:
```suggestion
**DroneCore.Power** contains 4 FOC ESCs with motor identification feature able to be configured for all common BLDC/PMSM drone motors on the market and a telemetry interface providing real time data to the flight controller. Furthermore it provides battery voltage and current measurements as well as individual cell voltages.
```

**Review by nkruzan — COMMENTED** [review]
Some nitpicks. Thanks

File: common/source/docs/common-airvolute-DroneCore-Suite.rst (hunk 1 above)
Comment (lukaspalkovic) [reply common/source/docs/common-airvolute-DroneCore-Suite.rst:1]: OK, I'd prefer to have the page ready and then to add it to common-autopilots

File: common/source/docs/common-airvolute-DroneCore-Suite.rst (hunk 6 above)
Comment (lukaspalkovic) [reply common/source/docs/common-airvolute-DroneCore-Suite.rst:77]: as input from Radio, the same way as Hex aero carrier boards do. What do you expect here?

**Pull request merged**